tox.ini

```diff
@@ -16,7 +16,7 @@ setenv =
 # To pass arguments to pytest, use `tox [options] -- [pytest posargs]`.
 commands =
     pytest --cache-clear -vv src/tests {posargs}
-    mypy --config-file=tox.ini src/
```

Comment:
What test warning does this fix?

Reply:
Good point, contrary to the title this does not fix a warning. However it makes the usage of mypy less verbose outside of tox which I think is a good thing. Maybe this should be part of a separate PR that improves tool configurations? I could open a PR that moves all tool-specific configuration to the pyproject.toml and also changes the deprecated [pep257] section name to [pydocstyle].

Reply:
If the point is to simplify usage for people refusing to use the project's tooling, we should move `pytest` options here to `add-opts` below rather than here, and we should have some other way to run `black` and `isort` because those are too cumbersome too, clearly, right?

People using pycharm + pytest won't have `--clear-cache` automatically configured for them or `-vv`. 

The point of using `tox` for things like this is to simplify things.

Reply:
I often just run a single tool from the tox pipeline (e.g., mypy while fixing type hints) and find it cumbersome to look up the specific command line options in the tox.ini. I therefore put all configuration in configuration files and only have calls like `pytest src/tests` or `mypy src/` in the tox.ini. So yes, I think moving `--clear-cache` and `-vv` into the `add-opts` section would be great because it centralizes the configuration. Am I missing some drawback of doing so? If so, why is pytest's `-rw` option in `add-opts`?

What other way to call black and isort do you mean? Their configs are already in the pyproject.toml so I can simply run `black src/pydocstyle` without worrying about additional command line options.

I am not sure about Pycharm but setting up tools like black and isort with VS Code picks up the configuration from pyproject.toml.

Reply:
@sigmavirus24, I did point out in the PR description that this is not strictly related. I can move it to another PR if you want.

While I agree that `tox` simplifies things, I have similar concerns to @Mr-Pepe. When I made this PR, I was working on improving type annotations to make the code easier to understand. With the current `tox` configuration, there isn't a way to run just `mypy` by itself, and even if there were, there wouldn't be an easy workflow to trying different arguments.

I agree that moving as many options to a single place would make it easier for alternative development workflows to be done. `tox` would be the ultimate test in CI and ideally before making a PR, but for fast iteration during development, it's overkill.

Reply:
> we should have some other way to run `black` and `isort`

Running [`make format`](https://github.com/PyCQA/pydocstyle/blob/5a1c4d3a128ce662301a80532a578f37f8d5c3c3/Makefile#L3) is an existing alternative way to run `black` and `isort` that is already part of the repository.